python/example_code/lookoutvision/inference.py

```diff
@@ -95,50 +83,204 @@ def download_from_s3(s3_resource, photo):
 
         return local_file
 
+    @staticmethod
+    def reject_on_classification(image, prediction, confidence_limit):
+        """
+        Returns True if the anomaly confidence is greater than or equal to 
+        the supplied confidence limit.
+        :param image: The name of the image file that was analyzed.
+        :param prediction: The DetectAnomalyResult object returned from DetectAnomalies
+        :param confidence_limit: The minimum acceptable confidence. Float value between 0 and 1.
```

Comment:
Should "Float value between 0 and 1." be changed to “(Float 0-1).” to match above?